app/services/submission/SubmissionService.scala

```diff
@@ -226,7 +226,9 @@ class SubmissionService @Inject()(registrationRepository: VatSchemeRepository,
       optional("agentOrTransactor" -> agentOrTransactor),
       conditional(specialSituations.nonEmpty)("specialSituations" -> specialSituations),
       conditional(attachmentList.nonEmpty)("attachments" -> vatScheme.attachments.map(attachmentDetails => jsonObject(
-        "attachmentMethod" -> attachmentDetails.method.toString,
```

Comment:
You can just do .map(_.toString) here and let the json handle the option, believe it just turns None into a null field. As it's just a log we don't really need the error message on the off chance it actually gets triggered.